cmd/diff/main.go

```diff
@@ -0,0 +1,203 @@
+package main
```

Comment:
I don't think this is needed if we move the CLI `diff.go` to `cmd/notary/diff.go`?